linux_files/setup

```diff
@@ -790,6 +793,10 @@ EOF
     sudo addgroup docker
     sudo adduser ${USER} docker
 
+    # can we delete the 'sources' folder left in wslupath -H?
```

Comment:
Yes I don't realize that a sources folder is left

Reply:
Eh, I cannot find the sources folder. The only one is left is in the tmp folder which is deleted at the end